services/web/client/source/class/qxapp/component/filter/UIFilterController.js

```diff
@@ -0,0 +1,67 @@
+/* ************************************************************************
+
+   qxapp - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2019 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Ignacio Pascual (ignapas)
+
+************************************************************************ */
+
+/**
+ * GUI filter controller. Stores the state of all filters and dispatches it when it changes.
+ */
+qx.Class.define("qxapp.component.filter.UIFilterController", {
+  extend: qx.core.Object,
+  type: "singleton",
+
+  construct: function() {
```

Comment:
I think in that case you can just NOT have a constructor ... not sure though

Reply:
Maybe, but it doesn't bother right?

Reply:
haha ... THAT sounds like a very good argument for having code baggage ...